components/raftstore/src/store/config.rs

```diff
@@ -717,8 +719,8 @@ impl Config {
             .with_label_values(&["io_reschedule_hotpot_duration"])
             .set(self.io_reschedule_hotpot_duration.as_secs() as f64);
         CONFIG_RAFTSTORE_GAUGE
-            .with_label_values(&["raft_msg_flush_interval_us"])
-            .set(self.raft_msg_flush_interval_us as f64);
+            .with_label_values(&["raft_msg_flush_interval"])
+            .set(self.raft_msg_flush_interval.as_micros() as f64);
```

Comment:
I think you need a new method that returns seconds as f64.

Reply:
It seems to be supported: https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_secs_f64